contracts/BaseLaunchpeg.sol

```diff
@@ -397,43 +325,23 @@ abstract contract BaseLaunchpeg is
         emit PublicSaleEndTimeSet(_publicSaleEndTime);
     }
 
-    /// @notice Set the reveal batch size. Can only be set after
-    /// batch reveal has been initialized and before a batch has
-    /// been revealed.
-    /// @dev Only callable by owner
-    /// @param _revealBatchSize New reveal batch size
-    function setRevealBatchSize(uint256 _revealBatchSize)
+    /// @notice Set the withdraw AVAX start time.
+    /// @param _withdrawAVAXStartTime New public sale end time
+    function setWithdrawAVAXStartTime(uint256 _withdrawAVAXStartTime)
         external
         override
         onlyOwner
     {
-        _setRevealBatchSize(_revealBatchSize);
+        withdrawAVAXStartTime = _withdrawAVAXStartTime;
+        emit WithdrawAVAXStartTimeSet(_withdrawAVAXStartTime);
```

Comment:
It should revert on `_withdrawAVAXStartTime < block.timestamp`, I don't see any point to set it in the past